<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Starting Time:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1" colspan="2">
     <widget class="QCheckBox" name="checkBox_5">
      <property name="text">
       <string>status</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Exiting Station ID:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QTextEdit" name="textEdit_5"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Total Time:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QTextEdit" name="textEdit_2"/>
    </item>
    <item row="8" column="1" rowspan="2" colspan="2">
     <widget class="QCheckBox" name="checkBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </item>
    <item row="15" column="3" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Load Dataset</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="2">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="text">
       <string>lineID</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QTextEdit" name="textEdit_3"/>
    </item>
    <item row="15" column="7">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Plan the route</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1" colspan="2">
     <widget class="QCheckBox" name="checkBox_6">
      <property name="text">
       <string>userID</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Time Step:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Inflow/Outflow(1/0):</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QTextEdit" name="textEdit_4"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Ending Time:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1" colspan="2">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="text">
       <string>stationID</string>
      </property>
     </widget>
    </item>
    <item row="15" column="5" colspan="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Plot Traffic Flow</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Entering Station ID:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" rowspan="2" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Select your interest field</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QTextEdit" name="textEdit_8"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Station ID:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6" colspan="2">
     <widget class="QTextEdit" name="textEdit_7"/>
    </item>
    <item row="12" column="1" colspan="2">
     <widget class="QCheckBox" name="checkBox_4">
      <property name="text">
       <string>deviceID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="2" column="4" rowspan="6" colspan="4">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="0" column="6" colspan="2">
     <widget class="QTextEdit" name="textEdit_6"/>
    </item>
    <item row="15" column="1" colspan="2">
     <widget class="QCheckBox" name="checkBox_7">
      <property name="text">
       <string>payType</string>
      </property>
     </widget>
    </item>
    <item row="16" column="3" colspan="5">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
